udata/tests/api/test_topics_api.py

```diff
@@ -83,6 +84,11 @@ def test_topic_api_list(self):
         self.assertEqual(len(response.json["data"]), 1)
         self.assertIn(str(granularity_topic.id), [t["id"] for t in response.json["data"]])
 
+        response = self.get(url_for("api.topics", featured="true"))
+        self.assert200(response)
+        self.assertEqual(len(response.json["data"]), 1)
+        self.assertIn(str(featured_topic.id), [t["id"] for t in response.json["data"]])
+
```

Comment:
The way I see it, this should work but doesn't with `type=bool`
```suggestion

        response = self.get(url_for("api.topics", featured="false"))
        self.assert200(response)
        self.assertEqual(len(response.json["data"]), 7)
        self.assertIn(str(featured_topic.id), [t["id"] for t in response.json["data"]])
```

Reply:
I had to change the model for this to work :-/ https://github.com/opendatateam/udata/pull/3301/files#diff-f3da585bd02f3f0b4060a6980b1a0ff7467e10d7bb1a19aaf8a044273547b27bR24

Should I write a migration? On my machine, all Topics created from API seem to have a boolean value, but shell created ones have `None`.

Reply:
I'll write one.

Reply:
I guess I could force `featured=True`, save the objects, and then set `featured=False`? Since we don't care about featured Topics so far, it shouldn't be a problem. But it is ugly :-/

Reply:
Am I missing something?
On this branch, having a previously created topic without `featured` field, I can set `featured=False` successfully with `.update`.
```
In [2]: Topic.objects(featured__exists=False).count()
Out[2]: 1

In [3]: Topic.objects(featured__exists=False).update(featured=False)
Out[3]: 1

In [4]: Topic.objects(featured__exists=False).count()
Out[4]: 0
```

Reply:
And we've used a similar logic [in dataservices recently](https://github.com/opendatateam/udata/pull/3262/files#diff-8349b3199575575b59667edd4d444d44441ef9b2df958558b97b5c81a232f5de)